src/clj/harja/kyselyt/muutos_kyselyt.sql

```diff
@@ -198,6 +227,20 @@ UPDATE mhu_muutos_kulu
  WHERE muutos = :muutos
    AND kulu = :vanha-kulu;
 
+-- name: paivita-muutostyo-kohdistus!
+UPDATE kulu_kohdistus
+   SET muutos = :muutos
+ WHERE id = :kohdistus-id;
+
+-- name: tarkista-onko-muutoksella-kuluja-ennen-voimassa-paivaa
```

Comment:
Tästä saisi suoraan ulos booleanin, niin ei tarvitsisi bäkärin koodissa vastausta muuntaa eri muotoon.


```suggestion
-- name: onko-muutoksella-kuluja-ennen-voimassa-paivaa?
-- single?: true

SELECT EXISTS(SELECT k.id...);
```

Reply:
Jees, korjailin ton muiden mukana